src/Distributed/DistributedBoundary.cc

```diff
@@ -1034,10 +1034,10 @@ cullGhostNodes(const FieldList<Dimension, int>& flagSet,
   typedef typename Boundary<Dimension>::BoundaryNodes BoundaryNodes;
   const int procID = domainID();
 
-  typedef NodeListRegistrar<Dimension> Registrar;
-  Registrar& registrar = Registrar::instance();
-  CONTRACT_VAR(registrar);
-  REQUIRE((int)numNodesRemoved.size() == registrar.numNodeLists());
+  // typedef NodeListRegistrar<Dimension> Registrar;
+  // Registrar& registrar = Registrar::instance();
+  // CONTRACT_VAR(registrar);
+  // REQUIRE((int)numNodesRemoved.size() == registrar.numNodeLists());
```

Comment:
Any reason to keep this around?

Reply:
No, no reason to keep it around. I can remove it.